### PR TITLE
[DASH-2278] Fix search bar and filter from custom sockets registry detail view

### PR DESCRIPTION
**Pull request by wpasinski**
before:

zrzut ekranu 2017-01-03 o 17 41 09

after:

zrzut ekranu 2017-01-04 o 12 40 07
zrzut ekranu 2017-01-04 o 12 42 29

### CODE REVIEW

**Comment by adamwardecki** [comment]
- I'd keep the filters right below the search
- Putting the 'Found x results' into a notification bar isn't a good idea. It draws attention to this element, while the number of results is actually not that important and more of an additional info
- I like the lighter color
- S inside the hexagon should look differently (already discussed)

File: src/apps/SocketsRegistry/DetailsView/SocketDetailsView.js [hunk 1]
```diff
@@ -129,34 +130,42 @@ const SocketsRegistryDetailsView = React.createClass({
...
+        <Helmet title={title} />
         <SocketInstallDialog
           shouldRedirect={true}
           url={currentSocket && currentSocket.ymlUrl}
         />
+        <InnerToolbar
+          title={title}
+          backButton={true}
+          backFallback={handleBackClick}
+          backButtonTooltip="Go back to Sockets Registry List"
```
Comment (tomsonpl) [inline src/apps/SocketsRegistry/DetailsView/SocketDetailsView.js]: Sockets Registry == List , So I think word `List` is not needed

File: src/apps/SocketsRegistry/SocketsRegistryList.js [hunk 2]
```diff
@@ -130,58 +116,81 @@ const SocketsRegistryList = React.createClass({
...
+    const { isLoading, filter, filterBySyncano, term } = this.props;
+
+    if (!term || isLoading) {
       return (
         <RegistryEmptyView
           title="Supercharge your project with Sockets from the community"
-          description={`Think of Sockets Registry a package manager for Syncano Sockets. You can search for
-            Sockets created by community and add them to your projects.`}
-          src={socketImageDir}
+          description={`Think of Sockets Registry a package manager for Syncano Sockets. You can search for Sockets
```
Comment (tomsonpl) [inline src/apps/SocketsRegistry/SocketsRegistryList.js]: Think of Sockets Registry as a package manager     - sorry probably my previous mistake;p

**Comment by wpasinski** [comment]
@adamwardecki Screenshots are updated.

**Comment by tomsonpl** [comment]
I think that LIST should work differently. There is no option to see all sockets, and imho there should be. What do you think about showing list of all at the beginning, and the zerostate only when filter doesn't find keyword?

**Comment by wpasinski** [comment]
@tomsonpl It makes more sense to me too, but it's just about moving the search bar and add some styling. Anyone? Do we want to show all sockets at start?

**Comment by tomsonpl** [comment]
Or leave zero state at the beginning but later, when filter isn't on, show all sockets. Better?

**Comment by adamwardecki** [comment]
@wpasinski lgtm (in terms of the design)
@tomsonpl make a task for this

**Pull request merged**